packages/SwingSet/src/kernel/vatManager/manager-subprocess-xsnap.js

```diff
@@ -159,14 +161,20 @@ export function makeXsSubprocessFactory({
       throw new Error(`failed to setBundle: ${bundleReply}`);
     }
 
-    /** @type { (item: Tagged) => Promise<Tagged> } */
+    /** @type { (item: Tagged) => Promise<CrankResults> } */
     async function deliver(delivery) {
       parentLog(vatID, `sending delivery`, delivery);
       transcriptManager.startDispatch(delivery);
       const result = await issueTagged(['deliver', ...delivery]);
-      parentLog(vatID, `deliverDone`, result[0], result.length);
+      parentLog(vatID, `deliverDone`, result.reply[0], result.reply.length);
       transcriptManager.finishDispatch();
-      return result;
+      return result.reply;
+      /*
+      return {
+        reply: result.reply,
+        crankStats: { workerType: 'xsnap-0.0.0', usedMeter: result.meterStats },
```

Comment:
The purpose of this comment seems to be to document the type of the return. Any particular reason not to use jsdoc / typescript to do that?

Reply:
It was more of a TODO to describe what I wanted for a return result.